production-umkc.yml

```diff
@@ -26,6 +26,7 @@ services:
       - ./.envs/.production/.django
       - ./.envs/.production/.postgres
       - ./.envs/.production/.secrets
+      - ./.envs/.production/.umkc
```

Comment:
for some reason, I'm not finding .umkc in your production folder

Reply:
I really like how you did it for local, though, so I'd just copy that, then you're good to go

Reply:
weird, it was getting blocked by the gitignore. Should be good now!